_install-and-configure/configuring-opensearch/security-settings.md

```diff
@@ -0,0 +1,483 @@
+
+The Security plugin supports the following REST layer TLS key store and trust store settings:
+
+- `plugins.security.ssl.http.enabled` (Static): Whether to enable TLS on the REST layer. If enabled, only HTTPS is allowed. Optional. Default is `false`.
+
+- `plugins.security.ssl.http.keystore_type` (Static): The type of the key store file. Optional. Valid values are `JKS` or `PKCS12/PFX` Default is `JKS`.
+
+- `plugins.security.ssl.http.keystore_filepath` (Static): The path to the key store file, which must be under the `config` directory, specified using a relative path. Required.
+
+- `plugins.security.ssl.http.keystore_alias` (Static):  The key store alias name. Optional. Default is the first alias.
+
+- `plugins.security.ssl.http.keystore_password`: The key store password. Default is `changeit`.
+
+- `plugins.security.ssl.http.truststore_type`: The type of the trust store file. Optional. Valid values are `JKS` or `PKCS12/PFX` Default is `JKS`.
+
+- `plugins.security.ssl.http.truststore_filepath`: The path to the trust store file, which must be under the `config` directory, specified using a relative path. Required.
+
+- `plugins.security.ssl.http.truststore_alias` (Static): The trust store alias name. Optional. Default is all certificates.
+
+- `plugins.security.ssl.http.truststore_password` (Static): The trust store password. Default is `changeit`.
+
+For more information, see [REST layer TLS]({{site.url}}{{site.baseurl}}/security/configuration/tls/#rest-layer-tls-1).
+
+## OpenSSL settings
+
+The Security plugin supports the following OpenSSL settings:
+
+- `plugins.security.ssl.transport.enable_openssl_if_available` (Static): Enables OpenSSL on the transport layer if available. Optional. Default is `true`.
+
+- `plugins.security.ssl.http.enable_openssl_if_available` (Static): Enables OpenSSL on the REST layer if available. Optional. Default is `true`.
+
+For more information, see [OpenSSL]({{site.url}}{{site.baseurl}}/security/configuration/tls/#advanced-openssl).
+
+## X.509 PEM certificates and PKCS #8 keys---transport layer TLS settings
+
+The Security plugin supports the following transport layer TLS settings related to X.509 PEM certificates and PKCS #8 keys:
+
+- `plugins.security.ssl.transport.pemkey_filepath` (Static): The path to the certificate's key file (PKCS #8), which must be under the `config` directory, specified using a relative path. Required.
+
+- `plugins.security.ssl.transport.pemkey_password` (Static): The key password. Omit this setting if the key has no password. Optional.
+
+- `plugins.security.ssl.transport.pemcert_filepath` (Static): The path to the X.509 node certificate chain (PEM format), which must be under the `config` directory, specified using a relative path. Required.
+
+- `plugins.security.ssl.transport.pemtrustedcas_filepath` (Static): The path to the root CAs (PEM format), which must be under the `config` directory, specified using a relative path. Required.
```

Comment:
"certificate authority" is spelled out throughout this doc. Edited to follow that format.